pkg/infra/filestorage/db_filestorage.go

```diff
@@ -2,71 +2,103 @@ package filestorage
 
 import (
 	"context"
+	"crypto/md5"
+	"encoding/hex"
+
+	// can ignore because we don't need a cryptographically secure hash function
+	// sha1 low chance of collisions and better performance than sha256
+	// nolint:gosec
+	"crypto/sha1"
+	"fmt"
 	"strings"
 	"time"
 
 	"github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
 	"github.com/grafana/grafana/pkg/util/errutil"
 )
 
 type file struct {
-	Path             string    `xorm:"path"`
-	ParentFolderPath string    `xorm:"parent_folder_path"`
-	Contents         []byte    `xorm:"contents"`
-	Updated          time.Time `xorm:"updated"`
-	Created          time.Time `xorm:"created"`
-	Size             int64     `xorm:"size"`
-	MimeType         string    `xorm:"mime_type"`
+	Path                 string    `xorm:"path"`
+	PathHash             string    `xorm:"path_hash"`
+	ParentFolderPathHash string    `xorm:"parent_folder_path_hash"`
```

Comment:
can you describe what these path hashes and parent hashes are for? indexing?

Reply:
added the comments here: https://github.com/grafana/grafana/blob/378b9466aa17f1833f0e49be38e37b9f0b9e2ff6/pkg/services/sqlstore/migrations/db_file_storage.go#L11-L27

should I move them over?